net/acme-common/files/acme.init

```diff
@@ -79,6 +86,12 @@ load_options() {
 		log warn "Please set \"option validation_method $validation_method\"."
 	fi
 	export validation_method
+
+	config_get httpport "$section" httpport 80
+	export httpport
+
+	config_get tlsport "$section" tlsport 443
```

Comment:
Why not just have a single "listen_port" directive, and then turn it into the correct option to acme.sh based on the validation method? This would also simplify the NFT rule creation...